api/s2n.h

```diff
@@ -120,6 +121,8 @@ extern uint32_t s2n_client_hello_get_cipher_suites_length(struct s2n_client_hell
 extern uint32_t s2n_client_hello_get_cipher_suites(struct s2n_client_hello *ch, uint8_t *out, uint32_t max_length);
 extern uint32_t s2n_client_hello_get_extensions_length(struct s2n_client_hello *ch);
 extern uint32_t s2n_client_hello_get_extensions(struct s2n_client_hello *ch, uint8_t *out, uint32_t max_length);
+extern uint32_t s2n_client_hello_get_extension_length(struct s2n_client_hello *ch, s2n_tls_extension_type extension_type);
```

Comment:
You also need to update the USAGE-GUIDE.md

Reply:
Will update.